cypress/integration/options/verticalAxis.cy.js

```diff
@@ -0,0 +1,105 @@
+import { DIMENSION_ID_DATA, VIS_TYPE_COLUMN } from '@dhis2/analytics'
+
+import { openDimension } from '../../elements/DimensionsPanel'
+import {
+    selectDataElements,
+    clickDimensionModalUpdateButton,
+} from '../../elements/DimensionModal'
+import { goToStartPage } from '../../elements/StartScreen'
+import { expectVisualizationToBeVisible } from '../../elements/Chart'
+import { TEST_DATA_ELEMENTS } from '../../utils/data'
+import { clickMenuBarOptionsButton } from '../../elements/MenuBar'
+import {
+    clickOptionsModalUpdateButton,
+    clickOptionsTab,
+    enableVerticalAxisTitle,
+    expectVerticalAxisRangeMaxToBeValue,
+    expectVerticalAxisRangeMinToBeValue,
+    expectVerticalAxisTitleToBeValue,
+    OPTIONS_TAB_AXES,
+    setVerticalAxisRangeMaxValue,
+    setVerticalAxisRangeMinValue,
+    setVerticalAxisTitle,
+} from '../../elements/OptionsModal'
+import {
+    expectStoreConfigYAxisToHaveRangeMinValue,
+    expectStoreConfigYAxisToHaveRangeMaxValue,
+    expectStoreConfigYAxisToHaveTitleText,
+} from '../../utils/store'
+
+const dimensionId = DIMENSION_ID_DATA
```

Comment:
Are you creating a new constant because in the future this test will loop through other dimensions? (Same question with the `optionsTab` const on line 32.

Reply:
Yeah,  I've tried to keep things as dynamic and flexible as possible. You'll find similar things in a lot of places where I've extracted parts that could be changed or looped over in the future and sometimes I also loop through arrays with only a single item.